packages/ibm-products/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -47,6 +47,7 @@ const Columns = ({
     [columns, setColumnsObject]
   );
 
+  const filteredStickyColumn = columns?.filter((item) => !item.sticky)?.map(item=> item);
```

Comment:
@Ratheeshrajan 
Do we need this .map() after .filter() ?

Reply:
Sorry, my mistake. removed